apps/dash-web-trader/requirements.txt

```diff
@@ -0,0 +1,33 @@
+attrs==19.1.0
```

Comment:
Do we absolutely need all of these packages? Would be nice to trim it down to only the ones that are needed (e.g., I don't think `jupyter-core` is needed, but correct me if I'm wrong)